announcements/1.11.2.md

```diff
@@ -0,0 +1,27 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.11.2
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+ * Android security updates for September 2021 (see [Android Security Bulletin - August 2021](https://source.android.com/security/bulletin/2021-09-01) for more information)
+ * Android WebView has been updated to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html)
+ * The AMS node controller now synchronizes its internal state to disk more often to avoid becoming out of sync across restarts with actual running containers
+ * Client side virtual keyboard is now supported on Android 11
+ * The default instance type of an application in the dashboard is now selected base on available GPU support
+
+#### Bugs
+
+ * LP #1938761 AnboxWebView lost focus for unknown reasons and causes client side virtual keyboard hidden in the end
+ * LP #1940807 Failed to launch anbox sessions with WebRTC platform (drm backend)
+ * LP #1940853 anbox-cloud-dashboard-51 charm fails to deploy
+ * LP #1942677 Audio/Video recording is broken on anbox swrast platform
+ * AC-304 Dashboard reports "Could not get response from Anbox Stream Gateway"
+ * AC-303 Dashboard lists non active images in application form
+ * AC-342 Connecting second ADB server breaks existing one
+
+#### Upgrade instructions
+
+See the [general upgrade guide](https://anbox-cloud.io/docs/installation/upgrading-from-previous-versions) for instructions on how to update your Anbox Cloud deployment to the 1.11.2 release.
```

Comment:
```suggestion
See [Upgrade Anbox Cloud](https://anbox-cloud.io/docs/installation/upgrading-from-previous-versions) or [Upgrade Anbox Cloud Appliance](https://anbox-cloud.io/docs/howto/upgrade/upgrade-appliance) for instructions on how to update your Anbox Cloud deployment to the 1.11.2 release.
```

Reply:
I split up this guide into two. Not sure if this release is for both Anbox Cloud and the Appliance though?